apps/server/src/institutions/getInstitutionHandler.ts

```diff
@@ -4,6 +4,7 @@ import { resolveInstitutionAggregator } from "./institutionResolver";
 
 export const getInstitutionHandler = async (req: Request, res: Response) => {
   const ucpInstitutionId = req.params.institution_guid;
+  req.context.latestResolvedInstitutionId = ucpInstitutionId;
```

Comment:
What do you think about this strategy to get the ucpInstitutionId without needing to make updates to the connect widget? In my testing it seems to be getting the institution right every time.